src/native/libs/System.Security.Cryptography.Native/pal_x509.c

```diff
@@ -871,27 +871,29 @@ static OCSP_CERTID* MakeCertId(X509* subject, X509* issuer)
     return OCSP_cert_to_id(EVP_sha1(), subject, issuer);
 }
 
+static time_t GetIssuanceWindowStart()
+{
+    // time_t granularity is seconds, so subtract 4 days worth of seconds.
+    // The 4 day policy is based on the CA/Browser Forum Baseline Requirements
+    // (version 1.6.3) section 4.9.10 (On-Line Revocation Checking Requirements)
```

Comment:
These requirements have changed effective 30 Sep 2020. See https://cabforum.org/wp-content/uploads/CA-Browser-Forum-BR-1.8.2.pdf, Sec. 4.9.10.

Reply:
The new rules are certainly more complicated :smile:.  In the 1.8.2 version the longest period they allow before informing an OCSP server of a change is still 4 days (Subscriber item 4), so I think it's still a reasonable maximum age cache boundary.  We could certainly look at making our window match their current policy later, if you like.  (FWIW, this code just moved up.  It was originally written Feb 2019)